iOS/MMIWG2S/ViewControllers/FacesARViewController.swift

```diff
@@ -92,7 +92,8 @@ extension FacesARViewController: ARFaceViewerUIDelegate {
     func generateShareImage() -> UIImage? {
         let image = sceneView.snapshot()
         sceneView.session.pause()
-        return image
+        let resizedImage = image.compress(to: 1000)
+        return resizedImage
```

Comment:
Do we want to return the resized image in `generateShareImage()`? i.e. would we want to give them the full sized image to share to their own social media and only compress the image sent to our db? Or compress both?

Reply:
Oh that's a great point. Yeah let's only compress what gets sent to the database. I'll check android as well on this thanks!